vpplink/types/vxlan.go

```diff
@@ -26,4 +28,5 @@ type VXLanTunnel struct {
 	DstPort        uint16
 	Vni            uint32
 	DecapNextIndex uint32
+	SwIfIndex      interface_types.InterfaceIndex
```

Comment:
Nice, we could also patch `AddVXLanTunnel` so that it sets this value before returning